libSetReplace compilation fails on 32-bit Windows

### The problem
Compilation fails on 32-bit Windows (on internal Wolfram Research build server) with the following error:
```
C:\BuildAgent\work\95e96ae48e9bf59e\SetReplace\libSetReplace\SetReplace.cpp(147):
  error C2664:
  'void SetReplace::putSet::<lambda_9875d9e26879c8d822293384ec7d6f4f>::operator
  ()(const std::vector<SetReplace::RuleID,std::allocator<int>> &) const': cannot
  convert argument 1 from 'const SetReplace::AtomsVector' to
  'const std::vector<SetReplace::RuleID,std::allocator<int>> &'
C:\BuildAgent\work\95e96ae48e9bf59e\SetReplace\libSetReplace\SetReplace.cpp(147):
  note: Reason: cannot convert from 'const SetReplace::AtomsVector' to
  'const std::vector<SetReplace::RuleID,std::allocator<int>>'
C:\BuildAgent\work\95e96ae48e9bf59e\SetReplace\libSetReplace\SetReplace.cpp(147):
  note: No user-defined-conversion operator available that can perform this conversion,
  or the operator cannot be called
```

The build log: [Windows_Build_128.log](https://github.com/maxitg/SetReplace/files/4644240/Windows_Build_128.log)


### Expected behavior
Compilation should work on Windows.

### Version
```wl
In[] := $SetReplaceGitSHA
```
6bdcf6c062b17a225015d4efbb7be2f46fccb9f7

--- a/libSetReplace/SetReplace.cpp
+++ b/libSetReplace/SetReplace.cpp
@@ -144,7 +144,8 @@ namespace SetReplace {
                         static_cast<mint>(atomsPointer)});
 
         for (const auto& expression : expressions) {
-            appendToTensor(expression.atoms);
+            // Cannot do static_cast due to 32-bit Windows support
+            appendToTensor(std::vector<mint>(expression.atoms.begin(), expression.atoms.end()));
         }
         
         return output;

--- a/scripts/buildInit.wl
+++ b/scripts/buildInit.wl
@@ -82,8 +82,11 @@ updateVersion[] /; Names["GitLink`*"] =!= {} := Module[{
   Check[
     versionInformation = Import[FileNameJoin[{$repoRoot, "scripts", "version.wl"}]];
     gitRepo = GitOpen[$repoRoot];
+    If[$internalBuildQ, GitFetch[gitRepo, "origin"]];
     minorVersionNumber = Max[0, Length[GitRange[
-      gitRepo, Except[versionInformation["Checkpoint"]], GitMergeBase[gitRepo, "HEAD", "master"]]] - 1];
+      gitRepo,
+      Except[versionInformation["Checkpoint"]],
+      GitMergeBase[gitRepo, "HEAD", If[$internalBuildQ, "origin/master", "master"]]]] - 1];
     pacletInfoFilename = FileNameJoin[{$buildDirectory, "PacletInfo.m"}];
     pacletInfo = Association @@ Import[pacletInfoFilename];
     versionString = pacletInfo[Version] <> "." <> ToString[minorVersionNumber];,

